src/refract/JSONSchemaVisitor.cc

```diff
@@ -575,40 +578,43 @@ namespace refract
         return ss.str();
     }
 
-    void JSONSchemaVisitor::processMembers(std::vector<refract::IElement*>::const_iterator begin,
-        std::vector<refract::IElement*>::const_iterator end,
+    void JSONSchemaVisitor::processMembers(const std::vector<refract::IElement*>& members,
         ArrayElement::ValueType& reqVals,
         std::vector<MemberElement*>& varProps,
         RefractElements& oneOfMembers,
         ObjectElement* o)
     {
-        for (auto it = begin; it != end; ++it) {
-            if (!*it) {
+        std::set<std::string> required;
+
+        for (const auto& member: members) {
+            assert(member);
+            if (!member) {
```

Comment:
This conditional is always entered. Otherwise line 590 would assert.

Reply:
Agree, but I do not want to crash drafter with `nullptr` on production

Reply:
Drafter wouldn't necessarily crash. It would yield to undefined behaviour, which is even worse. Nevertheless, the assert makes the branch impossible to unit test, so either choose an assert, or a conditional.

Reply:
And if you use a conditional, that path is definitely untested without unit tests for JSONSchemaVisitor.

Reply:
In general, there can be nullptr.
Then I will remove assert

Reply:
There is problem with testing, it is private function. We need bigger refactoring for this.

Reply:
At first look, it seem it is independent function and it can be isolated out of class

Reply:
So it use members `fixed`, `fixedType`, `pDefs`.
Will try to fix it, but not sure if able finish before sprint end